e2e/scenes/dashboard-suite/set-options-from-ui.spec.ts

```diff
@@ -137,6 +137,7 @@ describe('Variables - Set options from ui', () => {
     cy.intercept({ pathname: '/api/ds/query' }).as('query');
 
     cy.wait('@query');
+    cy.wait(300);
```

Comment:
Can't we wait for a specific element here? Usually, fixed times create flakiness depending on the network speed and instance performance. Also, they add extra time to the E2E execution which is suboptimal.

Reply:
@ivanortegaalba  I would really want to do this as well but could not identify anything to anchor on. This wait is the least amount of time needed and is to process the result of query and update the UI (I think this is related to the fact that variables no longer block rendering in scenes). But at the same time I have no clue why only 1 out of 3 tests need this adjustment 🤔

Reply:
Can't you wait for the loading bar to disappear?